argo/base/params.env

```diff
@@ -1,6 +1,7 @@
 namespace=
 executorImage=argoproj/argoexec:v2.3.0
 containerRuntimeExecutor=docker
+#containerRuntimeExecutor=k8sapi
```

Comment:
Can we delete this commented out code?

Reply:
Yes, I was trying to show options for runtime executor, otherwise its not clear what the options are and where to change them. The option is to put this into docs

Reply:
Do our params.env files support comments?
If you want to do it as a comment please include a more descriptive comment e.g.

# If you want to use a different executor change containerRuntimeExeccutor
# Possible values docker, k8sapi, etc...

Reply:
Just remove it

Reply:
It looks like the commented code is still hre.

p.s Apologies for the caps that was an accident